packages/cli/src/lib/configureStorybook.js

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs-extra'
+
+import { getPaths } from '.'
+
+/**
+ * Configure Storybook for the given template by creating a custom preview config
+ */
+export default function configureStorybook({ force }, newStorybookPreview) {
+  const { storybookPreviewConfig } = getPaths().web
```

Comment:
```suggestion
  const storybookPreviewConfigPath = getPaths().web.storybookPreviewConfig
```

I think adding `Path` to the end of the variable name makes it more obvious what this is when it's used later in the code